scripts/reload-clients

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+import argparse
+import configparser
+import logging
+import os
+import sys
+import time
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from scripts.lib.setup_path import setup_path
+
+setup_path()
+
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util import Retry
+
+from scripts.lib.zulip_tools import get_config, get_config_file, get_tornado_ports
+
+config_file = get_config_file()
+reload_rate = int(
+    get_config(
+        config_file,
+        "application_server",
+        "client_reload_rate",
+        "50",
```

Comment:
I'm not entirely sure what a good default reload rate is; I think 50/s (or 250 per 5s batch) is pretty fast; maybe that's OK? I figure we can tune this after merging in any case, given that the previous behavior in `main` was to do them all at once, it won't be worse.

Reply:
I did some eyeballing of this based on having Zulip Cloud finish in finite time, and that the current value is "infinity" so anything lower than that is notably slower.